server/src/main/java/keywhiz/service/daos/SecretDAO.java

```diff
@@ -109,6 +124,8 @@ public long createSecret(String name, String encryptedSecret, String hmac,
       SecretContentDAO secretContentDAO = secretContentDAOFactory.using(configuration);
       SecretSeriesDAO secretSeriesDAO = secretSeriesDAOFactory.using(configuration);
 
+      Long ownerId = getOwnerId(configuration, owner);
+
       Optional<SecretSeries> secretSeries = secretSeriesDAO.getSecretSeriesByName(name);
       long secretId;
       if (secretSeries.isPresent()) {
```

Comment:
Our API deals with owners via their names, but internally we need to record the ID (a long), so we introduce a lookup method to perform that translation.